src/UI/Art/SpriteArtist/SpriteUIUtils.java

```diff
@@ -108,6 +117,51 @@ public static BufferedImage makeTextMenu(Color bg, Color frame, Color focus, Arr
     return menuImage;
   }
 
+  /**
+   * Returns an image with the input string printed within the specified width, in normal text.
+   * @param reqWidth: Actual UI size in pixels that you want to fit the text into.
+   */
+  public static BufferedImage paintTextNormalized(String prose, int reqWidth)
```

Comment:
What does `Normalized` mean in this context? I feel like there's probably a better name for this function.

Reply:
"Normalized" meaning "fits in the specified width"

Reply:
`paintTextInFrame`? `drawTextToWidth`? `drawTextWithinSpecifiedBoundaryConstraints`?

Reply:
Hmm. Can I just change it in the `actionRework` branch to avoid conflicts?

Reply:
I suppose.